doc/src/manual/interfaces.md

```diff
@@ -322,8 +344,8 @@ forms, in most case you only need to specialize the 3-argument form.) For this t
 that `SparseArray` is mutable (supports `setindex!`). [`similar()`](@ref) is also used to allocate result
 arrays for arithmetic on `AbstractArrays`, for instance:
 
-```julia
-julia> A + 4
+```jldoctest squarevectype
+julia> A .+= 4
```

Comment:
Promoted the result to `Array`:

```julia
julia> A + 4
3×3 Array{Float64,2}:
 5.0   8.0  11.0
 6.0   9.0  12.0
 7.0  10.0  13.0
```

The call leads to this line https://github.com/JuliaLang/julia/blob/master/base/broadcast.jl#L37